src/BSeries.jl

```diff
@@ -1447,4 +1449,420 @@ include("latexify.jl")
     include("precompile.jl")
 end
 
+
+"""
+        is_energy_preserving(rka::RungeKuttaMethod,s)
+
+This function checks whether a Runge-Kutta method is energy 
+preserving for a given order s. 
+    Inputs: 
+        A: square matrix
+        b: vector of Butcher array
+        s: integer, the order
+    Output:
+        True/False
+"""
+function is_energy_preserving(rka::RungeKuttaMethod,s)
+#generate bseries 
+    series = bseries(rka, s)
+    #pass it to the main functoin 'is_energy_preserving'
+    is_energy_preserving(series)
+end
+
+"""
+
+        is_energy_preserving(series)::Bool
+This function checks whether the B-series `series` of a time integration
+method is energy preserving for Hamiltonian systems - up to the
+[`order`](@ref) of the `series`.
+"""
+#References:
+#This code is based on the Theorem 2 of the paper "Energy-Preserving 
+#Integrators and the Structure of B-series".
+# (link: https://link.springer.com/article/10.1007/s10208-010-9073-1).
+function is_energy_preserving(series)
```

Comment:
By inserting a comment here, the docstring is not connected to the function definition below.

Reply:
Solved: I quitted the comments and moved the docstring downwards.